packages/app-mobile/utils/setUpNotifications.ts

```diff
@@ -0,0 +1,30 @@
+import { NativeEventEmitter, NativeModules, Platform } from "react-native";
+
+export default async (dispatch: any) => {
+    if (Platform.OS === 'android') {
+        const { RNAlarmNotification } = NativeModules;
+        const RNAlarmEmitter = new NativeEventEmitter(RNAlarmNotification);
+
+        const handleNotification = async (notification: any) => {
```

Comment:
Could you create an interface for the notification object please? Just trying to avoid "any" as much as possible.